gradio/components.py

```diff
@@ -5330,6 +5331,101 @@ def as_example(self, input_data: str | None) -> str:
         return postprocessed if postprocessed else ""
 
 
+@document("change")
```

Comment:
as of #3420, you can remove the "change", and just have `@document()`, event listener documentation is automatically added